drivers/gles3/rasterizer_scene_gles3.cpp

```diff
@@ -4553,7 +4553,7 @@ void RasterizerSceneGLES3::render_scene(const Transform &p_cam_transform, const
 
 	_post_process(env, p_cam_projection);
 
-	if (false && shadow_atlas) {
+	if (false && shadow_atlas && storage->frame.current_rt) {
```

Comment:
It looks like all the code below was disabled in 8534ced22d when the above `_post_process` was added, so I guess this code should simply be deleted. CC @clayjohn @reduz.

Reply:
These blocks are used to display certain buffers (shadow atlas, reflections etc.). Deleting them would do no harm, but they would have to be re-written if we wanted to debug those features. That being said, GLES3 isn't going to be around for much longer so it may be best to just delete them and clean things up.

Reply:
Might be good to comment them out with `/* */` and add a comment explaining this code is for debugging purposes, and remove the `false &&` part of the conditions.